project/AkkaBuild.scala

```diff
@@ -125,6 +125,19 @@ object AkkaBuild extends Build {
     dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm", slf4j % "test->compile")
   ) configs (MultiJvm)
 
+  lazy val clusterTools = Project(
+    id = "akka-cluster-tools",
+    base = file("akka-cluster-tools"),
+    dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm")
+  ) configs (MultiJvm)
+  
+  lazy val clusterSharding = Project(
+    id = "akka-cluster-sharding",
+    base = file("akka-cluster-sharding"),
+    dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm", 
+        persistence % "compile;test->provided", clusterTools)
```

Comment:
I believe those are Ivy's "configurations", see here: http://ant.apache.org/ivy/history/latest-milestone/tutorial/conf.html 
and the `->`: http://ant.apache.org/ivy/history/latest-milestone/ivyfile/configurations.html

I do understand them the same way you do by the way.